src/patch.ts

```diff
@@ -10,14 +10,12 @@ import { VElement, VNode, VProps, VFlags } from './m';
  */
 /* istanbul ignore next */
 export const patchProps = (el: HTMLElement, oldProps: VProps = {}, newProps: VProps = {}): void => {
-  if (!oldProps && !newProps) return;
```

Comment:
As `oldProps` and `newProps` are by default objects with specific types we never will get **Falsy** values for them, so we never will run that `return`